platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/container/ContainerNode.java

```diff
@@ -255,8 +266,30 @@ public void onNext(final EventMessage value) {
 
                 @Override
                 public void onError(@NonNull final Throwable error) {
-                    final String message = String.format("gRPC error from node %s", selfId);
-                    fail(message, error);
+                    /*
+                     * After a call to killImmediately() the server forcibly closes the stream and the
+                     * client receives an INTERNAL error. This is expected and must *not* fail the test.
+                     * Only report unexpected errors that occur while the node is still running.
+                     */
+                    if (lifeCycle == RUNNING) {
```

Comment:
We should make the fields `lifeCycle`, `version`, and `platformStatus`  in `AbstractNode` `volatile`.